index.html

```diff
@@ -234,6 +253,7 @@ <h2 class="About">About
       <div class="eclipse12"></div>
       <div class="rectangle4"></div>
       <div class="eclipse3"></div>
+      <div class="half-circle-blue"></div>
       <h3>Contact me</h3>
```

Comment:
When you are adding the desktop view probably some bugs occurred. This one is one of them. Kindly fix the "Contact" heading, it needs to be positioned above the paragraph like before.
<img width="458" alt="Screenshot 2023-01-26 at 2 08 33 AM" src="https://user-images.githubusercontent.com/25887253/214715392-e8b29c07-2b58-481e-b8a0-a2b9990bf60c.png">